python/cudf/cudf/core/single_column_frame.py

```diff
@@ -325,3 +325,21 @@ def _make_operands_for_binop(
                 return NotImplemented
 
         return {result_name: (self._column, other, reflect, fill_value)}
+
+    def nunique(self, method: builtins.str = "sort", dropna: bool = True):
```

Comment:
@shwina should we keep this method parameter around? In practice we don't currently support any non-sort based methods, and this represents a deviation from the pandas API. Was there a pressing performance-related reason that we introduced this discrepancy in the first place?

Reply:
Yeah sounds like we can get rid of it safely. We should add a deprecation warning for it and eventually remove it. That can be done outside of this PR though.

Reply:
No need to make any changes in this PR, just leave the parameter as is. I discussed offline with @shwina and we will deprecate as needed in a follow-up.